ui/conductor/src/routes/ops/OpsNav.vue

```diff
@@ -160,10 +161,47 @@ const enabledMenuItems = computed(() => menuItems.value.filter((item) => uiConfi
  */
 const notificationEnabled = computed(() => uiConfig.pathEnabled('/ops/notifications'));
 
+/**
+ * Set the badge for the notification icon depending on the count or error
+ *
+ * @param {string} navItem
+ * @param {string} type
+ * @return {{color: string, value: number|string|null}}
+ */
+const setIconBadge = (navItem, type) => {
```

Comment:
Can you rename this function. `set` is used for setting a variable/property/data somewhere, this function doesn't set anything